src/vivarium_gates_iv_iron/constants/results.py

```diff
@@ -84,9 +84,9 @@
 
 POP_STATES = ("living", "dead", "tracked", "untracked")
 SEXES = ("female",)
-YEARS = tuple(range(2024, 2041))
+YEARS = tuple(range(2019, 2033))
 AGE_GROUPS = (
-    "5_to_9",
+    "0_to_10",
     "10_to_14",
```

Comment:
A little weird that our age group here overlaps with the next one, isn't it?

Reply:
Yes! Will update.